Pocket.xcodeproj/project.pbxproj

```diff
@@ -1007,7 +1007,7 @@
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = EX3VH4YFCH;
```

Comment:
is this because we do automatic provisioning now?

Reply:
yea

Reply:
there are other areas that I see `EX3VH4YFCH`, should we remove it there as well? Also getting the following errors:

![image](https://user-images.githubusercontent.com/6743397/218517658-5b8b1b10-247c-49c7-b860-cdeaa62ce822.png)